### ISSUE TITLE
Resizing works besides it being turned off on macOS

### ISSUE THREAD

**Issue by sphinxc0re** [emoon/rust_minifb#61]
Using this window config:
```rust
Window::new(
    "my window title",
    SCREEN_WIDTH,
    SCREEN_HEIGHT,
    WindowOptions {
        scale: Scale::X4,
        ..Default::default()
    }
)
```

**Comment by emoon** [comment]
Thanks for the report! I will take a look at it

**Comment by emoon** [comment]
Fixed in https://github.com/emoon/rust_minifb/commit/f181e2d67ce2602834ee52a0e7b1a8327beaac47 which will be included in the `0.15` release

**Comment by sphinxc0re** [comment]
Thanks!
